clients/python/test/e2e/test_notebooks.py

```diff
@@ -1,36 +1,37 @@
+import json
 import shutil
 import sys
 from pathlib import Path
 from tempfile import TemporaryDirectory
-from typing import Any, List
+from typing import Any, Dict, List, Optional, Set, Union
 
 import osparc
 import papermill as pm
 import pytest
 
-docs_dir: Path = Path(__file__).parent.parent.parent / "docs"
-all_notebooks: List[Path] = list(docs_dir.glob("*.ipynb"))
+# utilities --------------------------------------------------------------------------
 
+DOCS_DIR: Path = Path(__file__).parent.parent.parent / "docs"
+DATA_DIR: Path = Path(__file__).parent / "data"
+TUTORIAL_CLIENT_COMPATIBILITY_JSON: Path = (
+    DATA_DIR / "tutorial_client_compatibility.json"
+)
+API_DIR: Path = DOCS_DIR.parent.parent.parent / "api"
 
-def test_notebook_config(tmp_path: Path):
-    """Checks the jupyter environment is configured correctly"""
-    config_test_nb: Path = Path(__file__).parent / "data" / "config_test.ipynb"
-    assert config_test_nb.is_file()
-    test_run_notebooks(
-        tmp_path,
-        config_test_nb,
-        {
-            "expected_python_bin": sys.executable,
-            "expected_osparc_version": str(osparc.__version__),
-            "expected_osparc_file": osparc.__file__,
-        },
-    )
-    assert len(all_notebooks) > 0, f"Did not find any notebooks in {docs_dir}"
+assert DOCS_DIR.is_dir()
+assert DATA_DIR.is_dir()
+assert TUTORIAL_CLIENT_COMPATIBILITY_JSON.is_file()
+assert API_DIR.is_dir()
 
 
-@pytest.mark.parametrize("notebook", all_notebooks)
-def test_run_notebooks(tmp_path: Path, notebook: Path, params: dict[str, Any] = {}):
-    """Run all notebooks in the documentation"""
+def run_notebook(tmp_path: Path, notebook: Path, params: dict[str, Any] = {}):
```

Comment:
STYLE: every function not being fixtures or tests  in a `test/*/test_*.py` file are helpers. 
- The fixtures are decorated with `pytest.fixture`
- Tests are named `test_`
- (our convention) helpers use protected prefix, i.e. `def _run_notebook(...)`

Reply:
Makes sense. I will add a `_` in front of these.